package.json

```diff
@@ -69,6 +69,7 @@
     "react": "~16.0"
   },
   "dependencies": {
+    "@turf/turf": "5.1.6",
```

Comment:
Does `turf` have an integrated validation method, or is it possible to use the one of `jsts` (which is a dependency of `turf` AFAIK)

Reply:
I don't think turf.js has a validation method. We could a `jsts` as an extra dependency if we want to have validation in react-geo. We can't rely on the existence of `jsts` by using `turf.js` as the dependency might be remove.